test/inspector-cli/test-inspector-cli-pid.js

```diff
@@ -49,7 +49,7 @@ function launchTarget(...args) {
     // .then(() => cli.waitForPrompt())
     //
     // What we're diong for now:
```

Comment:
```suggestion
    // What we're doing for now:
```
nit: typo from an earlier change